crates/rooch/src/commands/statedb/commands/genesis_verify.rs

```diff
@@ -963,7 +963,7 @@ impl From<&Inscription> for InscriptionForComparison {
             offset: ins.location.offset,
             sequence_number: ins.sequence_number,
             inscription_number: ins.inscription_number,
-            is_curse: ins.is_curse,
+            is_curse: ins.is_cursed,
```

Comment:
all `is_curse` unify to  `is_cursed`, including sdk

Reply:
I make a new PR to update the sdk